internal/pkg/cli/pipeline_delete.go

```diff
@@ -118,8 +118,10 @@ func (o *deletePipelineOpts) Ask() error {
 
 // Execute deletes the secret and pipeline stack.
 func (o *deletePipelineOpts) Execute() error {
-	if err := o.deleteSecret(); err != nil {
-		return err
+	if o.PipelineSecret != "" {
```

Comment:
Do we not have tests for pipeline delete 😨

Reply:
Adding momentarily!

Reply:
Can we make this idempotent by wrapping a SecretNotFound error from the underlying client call in `deleteSecret` on L173?

Reply:
[discussed offline. moved conditional from `Execute()` into `deleteSecret()`]